Sources/SpeziLLMLocal/LLMLocalSession+Generate.swift

```diff
@@ -119,4 +124,43 @@ extension LLMLocalSession {
             state = .ready
         }
     }
+    
+    private func _mockGenerate(continuation: AsyncThrowingStream<String, any Error>.Continuation) async {
+        try? await Task.sleep(for: .seconds(1))
+        guard await !checkCancellation(on: continuation) else {
+            return
+        }
+        
+        continuation.yield("Mock ")
+        
+        try? await Task.sleep(for: .milliseconds(500))
+        guard await !checkCancellation(on: continuation) else {
+            return
+        }
+        continuation.yield("Message ")
+        
+        try? await Task.sleep(for: .milliseconds(500))
+        guard await !checkCancellation(on: continuation) else {
+            return
+        }
+        continuation.yield("from ")
+        
+        try? await Task.sleep(for: .milliseconds(500))
+        guard await !checkCancellation(on: continuation) else {
+            return
+        }
+        continuation.yield("SpeziLLM! ")
+        
+        try? await Task.sleep(for: .milliseconds(500))
+        guard await !checkCancellation(on: continuation) else {
+            return
+        }
+        continuation.yield("Using SpeziLLMLocal only works on physical devices.")
```

Comment:
Maybe we could do all of this in a loop to prevent repetition? I remember writing that code a long time ago and not liking it back then, we could clean it up a bit.

Reply:
Yeah, sounds good, done ✅